### ISSUE TITLE
Ability to write to stdout

### ISSUE THREAD

**Issue by lezsakdomi** [nonnymoose/xsr#26]
Use filename `-`

**Comment by lezsakdomi** [comment]
Alternative: `xsr /dev/stdout`

**Comment by lezsakdomi** [comment]
I discountinued working on because of #28 :)

**Comment by nonnymoose** [comment]
If the user does this, they need to be able to turn off other output too.

**Comment by lezsakdomi** [comment]
Yes, this bug requires a bit more refactor, but messages in a HTML file doesn't seem to be bad.

**Comment by lezsakdomi** [comment]
Requires solving #30 first

**Comment by lezsakdomi** [comment]
Just for reference: Probably breaks #49

**Comment by nonnymoose** [comment]
This should set verbosity to 0 to avoid breaking #49.
